test/runnable/test_cdvecfill.d

```diff
@@ -121,736 +121,1580 @@ struct Code(T_, int N_)
-        /* pop    rcx                     */ 0x59,
-        /* ret                            */ 0xc3,
-    ]),
-    Code!(long*, 32 / long.sizeof)([
-        /* push   rax                     */ 0x50,
-        /* vbroadcastsd ymm0,QWORD PTR [rdi] */ 0xc4, 0xe2, 0x7d, 0x19, 0x07,
-        /* pop    rcx                     */ 0x59,
-        /* ret                            */ 0xc3,
-    ]),
-    Code!(float, 16 / float.sizeof)([
-        /* push   rax                     */ 0x50,
-        /* vshufps xmm0,xmm0,xmm0,0x0     */ 0xc5, 0xf8, 0xc6, 0xc0, 0x00,
-        /* pop    rcx                     */ 0x59,
-        /* ret                            */ 0xc3,
-    ]),
-    Code!(float*, 16 / float.sizeof)([
-        /* push   rax                     */ 0x50,
-        /* vbroadcastss xmm0,DWORD PTR [rdi] */ 0xc4, 0xe2, 0x79, 0x18, 0x07,
-        /* pop    rcx                     */ 0x59,
-        /* ret                            */ 0xc3,
-    ]),
-    Code!(float, 32 / float.sizeof)([
-        /* push   rax                     */ 0x50,
-        /* vshufps ymm0,ymm0,ymm0,0x0     */ 0xc5, 0xfc, 0xc6, 0xc0, 0x00,
-        /* vinsertf128 ymm0,ymm0,xmm0,0x1 */ 0xc4, 0xe3, 0x7d, 0x18, 0xc0, 0x01,
-        /* pop    rcx                     */ 0x59,
-        /* ret                            */ 0xc3,
-    ]),
-    Code!(float*, 32 / float.sizeof)([
-        /* push   rax                     */ 0x50,
-        /* vbroadcastss ymm0,DWORD PTR [rdi] */ 0xc4, 0xe2, 0x7d, 0x18, 0x07,
-        /* pop    rcx                     */ 0x59,
-        /* ret                            */ 0xc3,
-    ]),
-    Code!(double, 16 / double.sizeof)([
-        /* push   rax                     */ 0x50,
-        /* vunpcklpd xmm0,xmm0,xmm0       */ 0xc5, 0xf9, 0x14, 0xc0,
-        /* pop    rcx                     */ 0x59,
-        /* ret                            */ 0xc3,
-    ]),
-    Code!(double*, 16 / double.sizeof)([
-        /* push   rax                     */ 0x50,
-        /* vmovsd xmm0,QWORD PTR [rdi]    */ 0xc5, 0xfb, 0x10, 0x07,
-        /* vunpcklpd xmm0,xmm0,xmm0       */ 0xc5, 0xf9, 0x14, 0xc0,
-        /* pop    rcx                     */ 0x59,
-        /* ret                            */ 0xc3,
-    ]),
-    Code!(double, 32 / double.sizeof)([
-        /* push   rax                     */ 0x50,
-        /* vunpcklpd xmm0,xmm0,xmm0       */ 0xc5, 0xf9, 0x14, 0xc0,
-        /* vinsertf128 ymm0,ymm0,xmm0,0x1 */ 0xc4, 0xe3, 0x7d, 0x18, 0xc0, 0x01,
-        /* pop    rcx                     */ 0x59,
-        /* ret                            */ 0xc3,
-        /* add    BYTE PTR [rax],al       */ 0x00, 0x00,
-    ]),
-    Code!(double*, 32 / double.sizeof)([
-        /* push   rax                     */ 0x50,
-        /* vbroadcastsd ymm0,QWORD PTR [rdi] */ 0xc4, 0xe2, 0x7d, 0x19, 0x07,
-        /* pop    rcx                     */ 0x59,
-        /* ret                            */ 0xc3,
-    ]),
-);
+version(D_PIC)
+{
```

Comment:
As mentioned at https://github.com/dlang/dmd/pull/7427#discussion_r156567743 running the update script is really useful and we shouldn't break that. Hence, I think it's best to make a copy of this file for `version(D_PIC)`?

Reply:
Specifically, which update script?

Reply:
The one which can be used to generate this output. Who did you get the `-fPIC` instructions?
Check the header of this file ;-)

Reply:
Ok, I change the update code to generate either PIC or non-PIC depending on what kind of machine it's run on.